vendor/assets/javascripts/behaviors/selectable_asset.js

```diff
@@ -0,0 +1,17 @@
+Cmsify.SelectableAsset = function(el) {
+  this.$el = $(el);
+  this.select = this.select.bind(this);
```

Comment:
@toastynerd - The reason this is required is because the method is called by the `click` event handler on line 4. Without binding `this`, then `this` would point to the object being clicked on instead of the instance of `Cmsify.SelectableAsset`.